package-res/resources/web/test/prompting/parameters/ParameterXmlParserSpec.js

```diff
@@ -75,6 +75,42 @@ define(['common-ui/prompting/parameters/ParameterXmlParser'], function (Paramete
         expect(paramDefn['errors']['null'][0]).toEqual("test global-error message");
         expect(paramDefn['errors']['test_parameter_name'][0]).toEqual("test error message");
       });
+
+      it("fails parse if invalid xml string is used", function () {
+        try {
+          parameterXmlParser.parseParameterXml();
+        } catch (e) {
+          expect(e).toBeDefined();
+        }
```

Comment:
@diogofscmariano Are you sure that expect(bar).toThrow(); (jasmine) does not work for this checks to avoid try - catch...?